bindata/etcd/pod.yaml

```diff
@@ -193,6 +193,26 @@ ${COMPUTED_ENV_VARS}
       periodSeconds: 5
       successThreshold: 1
       timeoutSeconds: 5
+    livenessProbe:
+      exec:
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -xe
+
+          # liveness probe uses a serialized get to an non-existant key (liveness) as a condition.
```

Comment:
I'm thinking about this.  We want to know we're failing, but I'm I want to be careful about indicating failure by crashlooping the thing we use to track what to run.  

Without a startup probe, this could result in a off-cycle etcd pods crashlooping and never starting at the same time to find quorum together, right?

Reply:
its the main reason I have avoided this is the fear of race but in some ENV such as single node under disruption we might want to help kubelet.